apps/explorer/src/app/App.tsx

```diff
@@ -14,67 +15,82 @@ import { growthbook, loadFeatures } from '../utils/growthbook';
 
 import styles from './App.module.css';
 
-const createQueryClient = () =>
-    new QueryClient({
-        defaultOptions: {
-            queries: {
-                refetchOnMount: false,
-                refetchOnWindowFocus: false,
-            },
+import { usePrevious } from '~/hooks/usePrevious';
+
+const queryClient = new QueryClient({
+    defaultOptions: {
+        queries: {
+            refetchOnMount: false,
```

Comment:
Why not keeping them both true to always be as up-to-sync as possible?

I am thinking if users lands on the page navigate to other pages and then go back to the initial page, would always see the stale data? Only refreshing the tab would make it fetch new data right?

Reply:
This is a great question. I have these disabled because the default stale time is 0 (i.e. everything is considered stale) which means this is super aggressive about fetching data. I think we can instead define a sane default stale time that lets us re-enable this (and we can also fine-tune at a query level when needed).

Reply:
I updated to add a default stale time and re-enable the refetching that was happening.